specification/resources/resource-manager/Microsoft.Authorization/stable/2019-09-01/policyAssignments.json

```diff
@@ -600,21 +600,27 @@
           }
         }
       },
-      "description": "The policy assignment properties."
+      "description": "The policy assignment properties.",
+      "required": [
```

Comment:
Change property from optional to required is breaking change.

Reply:
@raych1, yeah I agree.  However, it has always been required in the API itself so updating the swagger will bring the failure up to client-side instead of a server-side 400 response.  If that isn't advisable I'll remove the "required" changes in this PR.  What do you recommend?

Reply:
It will break previous SDK and I would suggest to remove from this PR and change it in future new API version.